### PR TITLE
Radial menus use callbacks instead of event datums

### PR DESCRIPTION
**Pull request by DamianX**
My crusade against `/event` continues.

### CODE REVIEW

File: __DEFINES/callback.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Set a callback's thing_to_call to GLOBAL_PROC in order to call a global proc.
+#define GLOBAL_PROC "fuck you duny"
```
Comment (jknpj) [inline __DEFINES/callback.dm:2]: huh

Comment (DamianX) [reply __DEFINES/callback.dm:2]: It can be anything as long as it's not null or empty

File: __DEFINES/callback.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+#define CALLBACK new /datum/callback
```
Comment (ShiftyRail) [inline __DEFINES/callback.dm]: I'm not sure if hiding the fact that you're creating a datum is the right thing to do.

Comment (DamianX) [reply __DEFINES/callback.dm]: ok, let's not

Comment (DamianX) [reply __DEFINES/callback.dm]: Changed the path to /callback so it's still short, removed the macro

File: code/datums/gamemode/role/ninja.dm [hunk 3]
```diff
@@ -346,7 +342,7 @@
 			var/event/menu_event = new(owner = user)
```
Comment (DamianX) [inline code/datums/gamemode/role/ninja.dm]: I need to remove this, do not merge

**Pull request merged**